modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/exp/IgniteSqlFunctionsTest.java

```diff
@@ -242,4 +245,57 @@ public void testConvertDecimal(String input, int precision, int scale, String re
             assertThrowsSqlException(Sql.RUNTIME_ERR, "Numeric field overflow", convert::get);
         }
     }
+
+    /** Tests for ROUND(x) function. */
+    @Test
+    public void testRound() {
```

Comment:
let's add all possible numeric types to the test cases. The same for another variant of this function

Reply:
There only 4 variants of this functions - int, long, double and BigDecimal. shorts and byte are promoted to ints by calcite. 
I can add tests for all numeric types at the SQL level.

Reply:
Because `ROUND(1)` has type `ROUND(<int>)`.
if there is no ROUND(<int>) overload,  than `SELECT ROUND(1)` is going to rejected by the validator.

Reply:
Have you tried to delete this overload and run test?

Reply:
Fixed.